src/types.ts

```diff
@@ -10,36 +10,53 @@ export enum TransactionAddressType {
 }
 
 export type RegisterSignupProps = {
+  installationId: string
   addressCoordinates?: AddressCoordinates
   addressLine?: string
-  appId?: string
```

Comment:
`appId` was not documented on the api reference, so I removed it. The removal is safe thanks to the wildcard addition.